packages/@dcl/inspector/src/lib/data-layer/host/rpc-methods.ts

```diff
@@ -3,12 +3,43 @@ import { Entity, EntityMappingMode, IEngine, Composite, OnChangeFunction, Compos
 import { DataLayerRpcServer, FileSystemInterface } from '../types'
 import { getFilesInDirectory } from './fs-utils'
 import { dumpEngineToComposite, dumpEngineToCrdtCommands } from './utils/engine-to-composite'
-import { createFsCompositeProvider } from './utils/fs-composite-provider'
+import { CompositeManager, createFsCompositeProvider } from './utils/fs-composite-provider'
 import { stream } from './stream'
 import { FileOperation, initUndoRedo } from './undo-redo'
 import { minimalComposite } from '../client/feeded-local-fs'
 import upsertAsset from './upsert-asset'
 
+function setupEngineDump(
+  fs: FileSystemInterface,
+  engine: IEngine,
+  compositeProvider: CompositeManager,
+  compositePath: string
+) {
+  return function dumpEngineAndGetComposite(dump: boolean = true): CompositeDefinition {
+    // TODO: hardcoded for the moment
+    const composite = dumpEngineToComposite(engine, 'json')
```

Comment:
we are doing this on every tick even if we didnt change anything. Is that ok ? Previuosly we had the isDirty condition